elements/lisk-p2p/src/validation.ts

```diff
@@ -21,6 +21,7 @@ import {
 	InvalidRPCResponseError,
 } from './errors';
 
+import { differenceBy, differenceWith, isEqual } from 'lodash';
```

Comment:
we can require specific ones? `require('lodash.differenceBy')`. @jondubois @shuse2  I thought we wanted to remove the lodash dependency in future?

Reply:
76ff823369a57e4fe0a934c837b8503b27dccf24